controller/login.go

```diff
@@ -48,11 +48,12 @@ type LoginController struct {
 	auth          login.KeycloakOAuthService
 	tokenManager  token.Manager
 	configuration loginConfiguration
+	identities    account.IdentityRepository
```

Comment:
Comment: identities can induce to people to think about an array or list of identites. But it is a repository... maybe better to name it identityRepo or something that better describe it.